manifests/s/Samsung/PlayTouchLauncher/1.2.0.32/Samsung.PlayTouchLauncher.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.PlayTouchLauncher
+PackageVersion: 1.2.0.32
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=4902668&CDCttType=SW&ModelType=C&ModelName=XE700T1A-A02US&VPath=SW/201204/20120404155905545/Play_Touch_Launcher_1.2.0.32.ZIP
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: setup.exe
+  InstallerSha256: B87EC63825B263A29B1AA10166A65160C8A7F55FEB42E2A10A380282ECB66425
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /passive

```

```suggestion
  NestedInstallerType: burn
  NestedInstallerFiles:
  - RelativeFilePath: setup.exe
  InstallerSha256: B87EC63825B263A29B1AA10166A65160C8A7F55FEB42E2A10A380282ECB66425
```
Installer type is burn. Using exe will cause it to throw "different install technology" error during upgrade